CONTRIBUTING.md

```diff
@@ -1,79 +1,99 @@
+
+## Development Flow
+
+We operate in two week sprints that are planned, estimated and prioritised on [Zenhub](https://app.zenhub.com/workspaces/human-connection-nitro-5c0154ecc699f60fc92cf11f). All issues are also linked to and synced with [Github](https://github.com/Human-Connection/Human-Connection/issues). Look for the `good first issue` label if you're not sure where to start!
+
+We try to discuss all questions directly related to a feature or bug in the respective issue, in order to preserve it for the future and for other developers. We use discord for real-time communication.
+
+This is how we solve bugs and implement features, step by step:
+1. We find an issue we want to work on, usually during the sprint planning but as an open source contributor this can happen at any time.
+2. We communicate with the team to see if the issue is still available. (When you comment on an issue but don't get an answer there within 1-2 days try to mention @Human-Connection/hc-dev-team to make sure we check in.)
+3. We make sure we understand the issue in detail – what problem is it solving and how should it be implemented?
+4. We assign ourselves to the issue and move it to `In Progress` on [Zenhub](https://app.zenhub.com/workspaces/human-connection-nitro-5c0154ecc699f60fc92cf11f).
+5. We start working on it in a `new branch` and open a `pull request` prefixed with `[WIP]` (work in progress) to which we regularly push our changes.
+6. When questions come up we clarify them with the team (directly in the issue on Github).
+7. When we are happy with our work and our PR is passing all tests we remove the `[WIP]` from the PR description and ask for reviews (if you're not sure who to ask there is @Human-Connection/hc-dev-team which pings all core developers).
+8. We then incorporate the suggestions from the reviews into our work and once it has been approved it can be merged into master!
+
+Every pull request needs to:
+* fix an issue (if there is something you want to work on but there is no issue for it, create on first and discuss it with the team)
+* include tests for the code that is added or changed
+* pass all tests (linter, backend, frontend, end-to-end)
+* be approved by at least 1 developer who is not the owner of the PR (when more than 10 files were changed it needs 2 approvals)
+
+## The Team
+
+There are many volunteers all around the world helping us build this network and without their contributions we wouldn't be where we are today. Big thank you to all of you!
+
+You can see the core team behind Human Connection [on our website](https://human-connection.org/en/the-team/). On Github you will mostly run into our developers:
+* Robert (@roschaefer)
+* Matt (@mattwr18)
+* Wolle (@Tirokk)
+* Alex (@ogerly)
+* Alina (@alina-beck)
+* Martin (@datenbrei), our head of IT
+* and sometimes Dennis (@DennisHack), the founder of Human Connection
+
+## Meetings and Pair Programming Sessions
+
+Daily standup
+* every Monday–Friday 10:45 UTC
```

Comment:
Publishing those events on UTC means it's a reference. So, if Germany has its clocks shifted from CET to CEST, our meeting in local time will change from 11:45 CET to 10:45 CEST. Are we OK with that? @mattwr18 @datenbrei @Tirokk @ogerly @alina-beck ?

Reply:
Yes. Also I'd add the place to the standup information (Discord Voicechat -> Coneference Room)

Reply:
Times are now indicated in "german time" – it's not perfect but clear enough, I hope?